### PR TITLE
{lib}[GCCcore/12.2.0] FUSE v3.14.1

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
I believe it should be changed ASAP, before next release

File: easybuild/easyconfigs/f/FUSE/FUSE-3.14.1-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+homepage = 'https://github.com/libfuse/libfuse'
+description = "The reference implementation of the Linux FUSE (Filesystem in Userspace) interface"
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = ['https://github.com/libfuse/libfuse/releases/download/fuse-%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['126919d72b46b3e0eb58a9c6933a2a50c36f2ea69f61fe9e78bdba9f463ffa20']
+
+builddependencies = [
```
Comment (Micket) [inline easybuild/easyconfigs/f/FUSE/FUSE-3.14.1-GCCcore-12.2.0.eb:15]: add binutils

Comment (sassy-crick) [reply easybuild/easyconfigs/f/FUSE/FUSE-3.14.1-GCCcore-12.2.0.eb:15]: Done. Sorry that slipped through.

**Comment by sassy-crick** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
> > I'm a bit uncertain on the value of this; probably more of a OS level thing, but, I suppose there is no harm in having this.
> 
> > 
> 
> > But surely, this is a GCCcore level package if anything?
> 
> 
> 
> Yep, you are right. I used an old EasyConfig as template, my wrong. 
> 
> That raises the question what we should do with the merged PR #17754 ?
> 
> 

My wrong too, should have spotted that when reviewing the PR, but overlooked it...

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux Rocky Linux 8.6, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/b2dcbe2d8879b871b20650cf589f2b08 for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**